vagga.yaml

```diff
@@ -1,18 +1,24 @@
 minimum-vagga: v0.7.0
 containers:
-  pip-freeze:
+  pip-freeze-main:
+    setup:
+      - !Alpine v3.4
+      - !Py3Install
+        - pip
+        - 'Django >=1.9,<1.10'
+        - 'django-environ >=0.4,<0.5'
+      - !Sh pip freeze > requirements/main.txt
+  pip-freeze-production:
```

Comment:
You'd better combine it all into a single container build by running `pip uninstall -r <file>.txt` after producing each file.